src/Workspaces/CSharp/Portable/xlf/CSharpWorkspaceResources.de.xlf

```diff
@@ -52,6 +52,16 @@
         <target state="translated">Einstellungen für den Musterabgleich</target>
         <note />
       </trans-unit>
+      <trans-unit id="Remove_and_Sort_Usings">
+        <source>R&amp;emove and Sort Usings</source>
+        <target state="new">R&amp;emove and Sort Usings</target>
```

Comment:
looks like translations were lost.

Reply:
hmm.. a refactoring to move resources between projects...

Reply:
yeah, i wrote a tool to do this myself.  pita.

Reply:
note: you might be ok if the loc team automatically dedupes these.  @tmeschter do you know what happens here?

Reply:
It'll be fine. The loc team will restore it from their master database of translations.